CoEpi.xcodeproj/project.pbxproj

```diff
@@ -41,6 +41,31 @@
 		84FA01002427ED5E00318104 /* HomeViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 84FA00FE2427ED5E00318104 /* HomeViewController.swift */; };
 		84FA01012427ED5E00318104 /* HomeViewController.xib in Resources */ = {isa = PBXBuildFile; fileRef = 84FA00FF2427ED5E00318104 /* HomeViewController.xib */; };
 		CC71F7CAF64914F41BBE05C5 /* Pods_CoEpi_CoEpiUITests.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 4AB30FDD8E48688E1BF91F91 /* Pods_CoEpi_CoEpiUITests.framework */; };
+		CE98BE88242CE27D00EFB67A /* HealthQuizViewModel.swift in Sources */ = {isa = PBXBuildFile; fileRef = CE98BE87242CE27D00EFB67A /* HealthQuizViewModel.swift */; };
+		CE98BE8A242CE4E400EFB67A /* QuestionCell.swift in Sources */ = {isa = PBXBuildFile; fileRef = CE98BE89242CE4E400EFB67A /* QuestionCell.swift */; };
+		D48ADDC7242D315100E595D9 /* GradientView.swift in Sources */ = {isa = PBXBuildFile; fileRef = D48ADDC6242D315100E595D9 /* GradientView.swift */; };
+		D48ADDC8242D315100E595D9 /* GradientView.swift in Sources */ = {isa = PBXBuildFile; fileRef = D48ADDC6242D315100E595D9 /* GradientView.swift */; };
```

Comment:
Did GradientView get added multiple times?

Reply:
Looks like so for several o these? Strings, Constants, Assets, OnboardingButton, etc